scripts/procedures/migrations.ts

```diff
@@ -24,5 +24,5 @@ export const migration: T.ExpectedExports.migration = compat.migrations
               down: () => { throw new Error('Downgrades are prohibited below 10.8.9.1 due to service instabilities below 10.9.8.1') },
```

Comment:
typo: 10.9.8.1 presumably isn't out yet

Reply:
Good catch, but fwiw any downgrade below 10.8.9.2 shouldn't be possible as 10.8.9.2 was the first version that went to prod.

That said, I don't believe there is a situation where a user would see this typo in the UI.